pytket/phir/phirgen_parallel.py

```diff
@@ -36,16 +36,16 @@ def process_sub_commands(
         int, list[tk.Command]
     ] = {}  # be sure to order by group number into a list when returning
     qubits2groups = {}  # track the most recent group in which a qubit was used
-    # group numbers for each gate are incremented by 2 so they don't overlap
+    # group numbers for each gate are incremented by 3 so they don't overlap
     # and different gate types don't go in the same group
-    # RZ gates go in %3=1 groups, R1XY gates go in %3=1 groups,
+    # RZ gates go in %3=0 groups, R1XY gates go in %3=1 groups,
     # and all other gates will go in %3=2 groups
```

Comment:
```suggestion
    # RZ gates go in (mod 3)=0 groups, R1XY gates go in (mod 3)=1 groups,
    # and all other gates will go in (mod 3)=2 groups
```
Minor suggestion, perhaps easier to read this way.